components/developer/valgrind/Makefile

```diff
@@ -22,18 +22,19 @@
 #
 # Copyright (c) 2015-2017 Aurelien Larcher. All rights reserved.
 # Copyright (c) 2019 Michal Nowak
+# Copyright (c) 2021 David Stes
 #
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		valgrind
-COMPONENT_VERSION=	3.15.0
+COMPONENT_VERSION=	3.17.0
 COMPONENT_FMRI=		developer/debug/valgrind
 COMPONENT_CLASSIFICATION=Development/System
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.bz2
 COMPONENT_ARCHIVE_HASH=	\
-	sha256:417c7a9da8f60dd05698b3a7bc6002e4ef996f14c13f0ff96679a16873e78ab1
+	sha256:ad3aec668e813e40f238995f60796d9590eee64a16dff88421430630e69285a2
 COMPONENT_ARCHIVE_URL=	ftp://sourceware.org/pub/valgrind/$(COMPONENT_ARCHIVE)
 COMPONENT_PROJECT_URL =	http://valgrind.org
 COMPONENT_SUMMARY=	Valgrind: instrumentation framework and tools to detect memory and threading problems
```

Comment:
...replace
include $(WS_MAKE_RULES)/prep.mk
include $(WS_MAKE_RULES)/configure.mk
include $(WS_MAKE_RULES)/ips.mk
with
include $(WS_MAKE_RULES)/ips.mk
with
include $(WS_MAKE_RULES)/ips.mk
with
include $(WS_MAKE_RULES)/common.mk

and finally...